src/Faker/Provider/en_IN/DateTime.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace Faker\Provider;
+
+class DateTime extends \Faker\Provider\Base
+{
+    protected static $century = array('I','II','III','IV','V','VI','VII','VIII','IX','X','XI','XII','XIII','XIV','XV','XVI','XVII','XVIII','XIX','XX','XXI');
+
+    protected static function getMaxTimestamp($max = 'now')
+    {
+        if (is_numeric($max)) {
+            return (int) $max;
+        }
+
+        if ($max instanceof \DateTime) {
+            return $max->getTimestamp();
+        }
+
+        return strtotime(empty($max) ? 'now' : $max);
+    }
+
+    /**
+     * Get a timestamp between January 1, 1970 and now
+     *
+     * @param \DateTime|int|string $max maximum timestamp used as random end limit, default to "now"
+     * @return int
+     *
+     * @example 1061306726
+     */
+    public static function unixTime($max = 'now')
```

Comment:
No you don't need to override the methods that don't change. Have you heard of inheritance?